docs/api/api_blueprint.apib

```diff
@@ -5,7 +5,7 @@ HOST: http://nextgen.eventyay.com
 
 The Open Event API Server
 
-## JWT Auth Key [/auth/session?filter_parameters]
+## JWT Auth Key [/auth/session?filter-parameters]
```

Comment:
@niranjan94 this has been fixed in PR by @magdalenesuo [here](https://github.com/fossasia/open-event-orga-server/pull/3775). So can we merge this?